contrib/google.golang.org/grpc/client.go

```diff
@@ -29,6 +29,9 @@ func (cs *clientStream) RecvMsg(m interface{}) (err error) {
 			cs.cfg.clientServiceName(),
 			cs.cfg.analyticsRate,
 		)
+
+		span.SetTag(ext.SpanKind, ext.SpanKindClient)
```

Comment:
This method seems to be about the client receiving a message back from the server; the span there shouldn't be marked as kind:client. Maybe @rmfitzpatrick can help confirm.

Reply:
Right, I think it's for the best to only have functions that inject context into a carrier as denoting the client kind.